src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageWithPropertiesExtensions.cs

```diff
@@ -191,15 +190,15 @@ public static class LoggerConfigurationAzureTableStorageWithPropertiesExtensions
 
             try
             {
-                var credentials = new StorageCredentials(sharedAccessSignature);
+                StorageCredentials credentials = new StorageCredentials(sharedAccessSignature);
```

Comment:
lets keep using `var` where we can, can update variable name to storageCredentials instead